Source/Core/Core/PowerPC/JitInterface.cpp

```diff
@@ -218,6 +218,16 @@ namespace JitInterface
 		return jit->HandleFault(access_address, ctx);
 	}
 
+	bool HandleStackFault()
+	{
+		if (!jit)
+		{
+			return false;
+		}
+
```

Comment:
Not needed yes, but this doesn't mean you must not put them there. Single-line statements with braces are ok (and recommended whenever it makes sense due to how the rest of the function/file looks like).